release_tester/selenium_ui_test/pages/analyzers_page.py

```diff
@@ -1112,6 +1114,397 @@ def add_new_analyzer(self, name, ui_data_dir=None):
         except TimeoutException:
             print("Error occurred!! required manual inspection.\n")
         print(f"Creating {name} completed successfully \n")
+
+        # --------------------here we are checking the properties of the created analyzer----------------------
+        if self.version_is_newer_than('3.11.99'):
+            try:
+                analyzer = f"//*[text()='_system::{name}']"
+                analyzer_sitem = self.locator_finder_by_xpath(analyzer)
+                if analyzer_sitem is None:
+                    print(f'This {analyzer_name} has never been created \n')
+                else:
+                    analyzer_sitem.click()
+                    time.sleep(1)
+
+                    # finding the ace editor using neighbor locators
+                    nearest_button = "//button[@class='jsoneditor-compact']"
+                    ace_locator = self.locator_finder_by_xpath(nearest_button)
+                    # Set x and y offset positions of adjacent element
+                    xOffset = 50
+                    yOffset = 50
+                    # Performs mouse move action onto the element
+                    actions = ActionChains(self.webdriver).move_to_element_with_offset(ace_locator, xOffset, yOffset)
+                    actions.click()
+                    actions.key_down(Keys.CONTROL).send_keys("a").key_up(Keys.CONTROL).key_down(Keys.CONTROL).send_keys(
+                        "c").key_up(Keys.CONTROL)
+                    actions.perform()  # Execute the sequence of actions
+
+                    # Retrieve text content from the clipboard using Pyperclip
+                    actual_properties = ''.join(str(pyperclip.paste()).split())
+                    # Get expected properties based on analyzer name
+                    expected_properties = ''.join(str(self.generate_expected_properties(name, ui_data_dir)).split())
+                    # Assert that the copied text matches the expected text
+                    try:
+                        assert actual_properties == expected_properties, "Text does not match the expected text \n"
+                    except AssertionError as ex:
+                        print("actual_properties: ", actual_properties)
+                        print("expected_properties: ", expected_properties)
+                        raise AssertionError("Text does not match the expected text") from ex
+                    else:
+                        print("Text matches the expected text. \n")
+
+            except TimeoutException as ex:
+                print(f'Failed to parse properties from the {name} and the error is: {ex} \n')
```

Comment:
Why we catch and do not throw again? Potentially, it will hide errors from us

Reply:
fixed.